### PR TITLE
[13.0][MIG] website_cookie_notice: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
**Superseded** #681 

cc @tecnativa TT19839

### CODE REVIEW

**Review by chienandalu — CHANGES_REQUESTED** [review]
- pre-commit failing
- When I click the "Ok" button, the cookies message doesn't go away

**Comment by Tardo** [comment]
Fixed, thanks! the pre-commit warnings are no related with this module... but i added another "[IMP] black, isort, prettier" commit to solve it.

**Review by chienandalu — COMMENTED** [review]
I observe this weird behavior:
cookies-message-reappears
Shouldn't the preference be persistent?

**Comment by Tardo** [comment]
Ummm yes by code this is store on the session cookie... i don't know if is best store it on a persistent cookie. What happens with public users on a shared computer if we store it on a persistent cookie?

**Comment by pedrobaeza** [comment]
Better to keep it this way as Odoo integrates it in v14

**Review by chienandalu — APPROVED** [review]
Ok, fine. It's annoying but I guess it's been like that until now

**Review by pedrobaeza — APPROVED** [review]

File: .isort.cfg [hunk 1]
```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-default_section=THIRDPARTY
+known_third_party=setuptools
```
Comment (pedrobaeza) [inline .isort.cfg]: Oh, wait, you mustn't change this!! Have you rebased?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by Tardo** [comment]
Ok, hope now is ready.

**Comment by Tardo** [comment]
This PR need reviews

**Comment by chienandalu** [comment]
I guess only @pedrobaeza blessing is left

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**